app/controllers/thread_controller.php

```diff
@@ -62,19 +65,53 @@ public function create()
     {
         $thread = new Thread;
         $comment = new Comment;
-        $page = Param::get('page_next', 'create');
+        $page = Param::get(Thread::PAGE_NEXT, Thread::PAGE_CREATE);
+        $user_id = Session::get('id');
 
         switch ($page) {
-            case 'create':
+            case Thread::PAGE_CREATE:
                 break;
-            case 'create_end':
+            case Thread::PAGE_CREATE_END:
                 $thread->title = Param::get('title');
+                $thread->user_id = $user_id;
                 $comment->username = Param::get('username');
                 $comment->body = Param::get('body');
+
+                $thread->validate();
+                $comment->validate();
+                if($thread->hasError() || $comment->hasError())
+                {
```

Comment:
Please follow the format for the if-else conditions :bow: 
```
if (condition) {
    ...
} else {
    ...
}
```
Please apply to all others :dart: